### PR TITLE
feat(bridge): Remove tag input field for creating a sequence

### LINKED ISSUE #7704: Bridge: Remove tag input field for delivery sequence
## Technical Details

The field `tag` in the delivery sequence should be removed and only the input field `image` should be used.

## Acceptance Criteria

- [ ] The `tag` input field is removed
- [ ] The `image` input field can allow `:` as character (not mandatory to be there though)
- [ ] The watermark text for the `image` input field shows `keptnexamples/simplenodeservice:5.0.4`

## DoD

- [ ] Unit/UI tests are adapted

### PR DESCRIPTION
**Pull request by heinzburgstaller**
## This PR
- Removes input tag
- Change validation
- Adapt unit test
- Adapt integration test
- Add form util function and unit test for it

### Related Issues
Resolves #7704 

### How to test
Ensure that the environment variable `KEPTN_INSTALLATION_TYPE` contains `CONTINUOUS_DELIVERY`
Steps:
1. Trigger sequence
2. Choose delivery
3. Image fields is now full size and tag is removed

### Screenshot
image

### CODE REVIEW

File: bridge/cypress/integration/trigger-sequence.spec.ts [hunk 1]
```diff
@@ -91,9 +91,7 @@ describe('Trigger a sequence', () => {
       .typeDeliveryValues('": "val2"}')
       .assertDeliveryValuesErrorExists(false)
       .assertTriggerSequenceEnabled(false)
-      .typeDeliveryImage('docker.io/keptn')
-      .assertTriggerSequenceEnabled(false)
-      .typeDeliveryTag('v0.1.2')
```
Comment (ermin-muratovic) [inline bridge/cypress/integration/trigger-sequence.spec.ts:96]: This method could also be removed from `TriggerSequenceSubPage.ts`

**Review by ermin-muratovic — CHANGES_REQUESTED** [review]

**Comment by thisthat** [comment]
Thank you @heinzburgstaller for your contribution 🙌 Could you please attach a screenshot how the UI looks like to the PR description?

**Comment by heinzburgstaller** [comment]
> Thank you @heinzburgstaller for your contribution raised_hands Could you please attach a screenshot how the UI looks like to the PR description?

Done

**Review by ermin-muratovic — DISMISSED** [review]
lgtm. 
can't reproduce the failed UI test. maybe a flaky test, but not a concern of this issue.

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**